charts/postgresql/templates/statefulset-slaves.yaml

```
@@ -91,7 +92,8 @@ spec:
         {{- if .Values.securityContext.enabled }}
         securityContext:
           runAsUser: {{ .Values.securityContext.runAsUser }}
-        {{- end }}
+          runAsNonRoot: {{ .Values.securityContext.runAsNonRoot }}
+  {{- end }}
```

This should be aligned with the `if` block, which I think is:
```suggestion
        {{- end }}
```